pkg/services/secrets/kvstore/kvstore.go

```diff
@@ -80,6 +81,9 @@ type SecretsKVStore interface {
 	Del(ctx context.Context, orgId int64, namespace string, typ string) error
 	Keys(ctx context.Context, orgId int64, namespace string, typ string) ([]Key, error)
 	Rename(ctx context.Context, orgId int64, namespace string, typ string, newNamespace string) error
+	GetAll(ctx context.Context) ([]Item, error)
+	Fallback() SecretsKVStore
```

Comment:
What is your opinion on using a combined getter/setter here in order to avoid making the interface too bulky?

Interface would look like:
```
Fallback(fb SecretsKVStore) SecretsKVStore
```

Impl would look like:
```
func (kv *SecretsKVStore) Fallback(fb SecretsKVStore) SecretsKVStore {
    if fb != nil {
        kv.fallback = fb
    }
    return kv.fallback
}
```

Reply:
I was planning on keeping them separate so we can SetFallback(nil) to disable it.

I think it's a bit weird to have a getter-setter function on Golang since there are no default arguments, in TypeScript it would make sense since Fallback() would be the getter, Fallback(null) would be the "unsetter" and Fallback(fb) could be a setter, but in Golang we would have to use Fallback(nil) as a getter and Fallback(fb) as a setter, needing another function to work as "unsetter" like Remove... or DisableFallback() so in the end we would still have 2 functions.

Reply:
Yeah, that makes sense.